server/parsec/components/postgresql/invite.py

```diff
@@ -1376,13 +1374,13 @@ async def test_dump_all_invitations(
 
     # Helpers
 
-    async def get_active_attempt(
+    async def get_greeting_session(
         self,
         conn: AsyncpgConnection,
         organization_id: OrganizationID,
         greeter: UserID,
         invitation_internal_id: int,
-    ) -> tuple[int, GreetingAttemptID]:
+    ) -> int:
```

Comment:
Here it's an int because a greeting session only has an id in the DB ?

It's a matter of personal preference, but I'd add a comment or a type alias to make that explicit

Reply:
I wanted to add a type alias but I forgot your comment and merged the PR. I'll add it later if I have to come back to fix some stuff